test/Makefile

```diff
@@ -66,8 +89,9 @@ format_unittest: $(GTEST_LIB) format_unittest.o $(SRC_DIR)/format.o
 	$(CXX) $(LDFLAGS) $^ $(LDLIBS) -o $@
 base64_unittest: $(GTEST_LIB) base64_unittest.o $(SRC_DIR)/base64.o
 	$(CXX) $(LDFLAGS) $^ $(LDLIBS) -o $@
-
+configuration_unittest: $(GTEST_LIB) configuration_unittest.o $(SRC_DIR)/configuration.o
```

Comment:
Is this not supposed to also depend on `$(YAML_CPP_LIBS)`?

Reply:
I believe it's still relevant -- if you don't add that dependency, then `make test` in a fresh checkout will fail to link (because it won't rebuild `$(YAML_CPP_LIBS)`).

Reply:
I think you are correct - fixed